src/__tests__/Search.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../conponents/Search";
+
+
+describe("Search", () => {
+  const { asFragment } = render(<Search />);
+
+  it("renders correctly", () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
1 of your snapshots is failing and 2 are obsolete. Make sure to check that your snapshots are what you expect them to be and then update them accordingly